azure-pipelines.yml

```diff
@@ -20,6 +18,8 @@ jobs:
         python.version: '3.8'
       Python39:
         python.version: '3.9'
+      Python310:
```

Comment:
Could you comment all these 3.10 tests since they fail at the moment?
Ideally, you could have some word here about the fact that this should be resolved by the end of 2021 and provide the link to TF forum you gave in the comments?

Reply:
I'll separate the python 3.10 things into it's own PR that you can merge later